website/pages/_app.tsx

```diff
@@ -17,11 +17,11 @@ import {
 import { CookieBanner, GlobalFooter, GlobalHeader } from '@edgeandnode/go'
 
 import { supportedLocales, translations, useI18n } from '@/i18n'
-import { DocSearch } from '@/src/nextra-theme/components'
+import { DocSearch } from '@/src/components'
 
 import '@edgeandnode/gds/style.css'
 import '@docsearch/css'
-import '../src/app.css'
+import './globals.css'
```

Comment:
Just a preference thing?

Reply:
yeah, similar to structure like in `app` router, can do rollback of this change

Reply:
Ah, no, if that's how Next calls it by default, let's go with that. (By the way, does Nextra support the `app` router? If so, do you think we should switch to it?)

Reply:
Nextra v4 will support only `app` router + server side i18n. I did like 80% of the migration in January. After worked on GraphQL.org migration to Nextra and new features to Hive